src/http/apis/bankid/request-builder.ts

```diff
@@ -0,0 +1,42 @@
+import { IBankIdAuthRequest } from "../../../bankid/types";
```

Comment:
Good move, moving these types of functions into their own context/file!

Reply:
thanks.